src/components/Sessions/SessionExpired.stories.js

```diff
@@ -0,0 +1,13 @@
+import {withRouter} from 'storybook-addon-react-router-v6';
+
+import SessionExpired from './SessionExpired';
+
+export default {
+  title: 'Views / Session Expired',
+  component: SessionExpired,
+  decorators: [withRouter],
+};
+
+export const Default = {
+  name: 'Session Expired',
+};
```

Comment:
I did some experimenting locally to get this story to work in non-debug mode too (which is happening in Chromatic, and the following code gets us there:

```suggestion
import {useEffect} from 'react';
import {withRouter} from 'storybook-addon-react-router-v6';

import {sessionExpiresAt} from 'api';
import useSessionTimeout from 'hooks/useSessionTimeout';
import {ConfigDecorator} from 'story-utils/decorators';

import SessionExpired from './SessionExpired';

const render = args => {
  const [expired] = useSessionTimeout();
  useEffect(() => {
    const now = new Date();
    sessionExpiresAt.setValue({expiry: now});
    return () => {
      sessionExpiresAt.setValue({expiry: null});
    };
  });
  if (!expired) {
    return 'Waiting for expiry...';
  }

  return <SessionExpired {...args} />;
};

export default {
  title: 'Views / Session Expired',
  component: SessionExpired,
  decorators: [ConfigDecorator, withRouter],
  parameters: {
    config: {
      debug: false, // force false in local dev-mode
    },
  },
};

export const Default = {
  name: 'Session Expired',
  render,
};

```

However, this will require some changes to `useSessionTimeout` too which I'll leave for you to figure out :wink: